src/modules/map/components/LocationPopupCopySlot.vue

```diff
@@ -14,7 +14,7 @@
 export default {
     props: {
         value: {
-            type: String,
+            type: [String, null, undefined],
```

Comment:
It would maybe be more elegant to say that `value` is not required, and it has a default value of `null`
```
{
   type: String,
   default: null,
}
```
And subsequently adapt the component (if need be) to manage a `null` value here

Or giving it a default value of `''`, so that if nothing (or `null` or `undefined`) is given, the prop has a valid default value

Reply:
Good idea 👍🏼